app/src/main/java/org/jellyfin/androidtv/ui/widget/Rating.kt

```diff
@@ -0,0 +1,73 @@
+package org.jellyfin.androidtv.ui.widget
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.Gravity
+import android.view.View
+import android.widget.ImageView
+import android.widget.RelativeLayout
+import android.widget.TextView
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.util.dp
+import kotlin.math.roundToInt
+import kotlin.properties.Delegates
+
+class Rating(context: Context, attrs: AttributeSet?) : RelativeLayout(context, attrs) {
+	private val attributes = context.theme.obtainStyledAttributes(attrs, R.styleable.Rating, 0, 0)
+
+	private val imageView = ImageView(context).apply {
```

Comment:
Why do you create the elements manually and not using a layout file?

Reply:
It's simple enough to not use a XML file. I hate XML files. If I don't need to use them I don't.